components/pedigree/resources/src/main/resources/pedigree/model/helpers.js

```diff
@@ -9,12 +9,9 @@
 define ([], function(){
 
     var Helpers = {};
-    
-    // Fixes bug introduced after utilizing requirejs where the global console variable becomes undefined.
-    if (!console) { var console = window.console; }
-    
-    // To allow debug code to run in IE7 && IE8
-    if (!window.console) { var console = {log: function() {}}; }
+
+    // To allow debug code to run in IE7 && IE8 && IE9
+    if (!window.console) { window.console = {log: function() {}}; }
 
     // For IE7 && IE8 again
```

Comment:
we now do, but since compatibility code is already there, why remove it. The issue at hand that this commit fixes - problems with console - affects IE9 as well. If at some point even view does not work in IE8 it will be disallowed on PT level (we do a warning right now but let user continue), but for now there is no hard reason to disallow viewing pedigrees in IE8